argoverse/evaluation/eval_forecasting.py

```diff
@@ -115,20 +115,9 @@ def get_displacement_errors_and_miss_rate(
 
         if forecasted_probabilities is not None:
             prob_n_misses.append(1.0 if curr_min_fde > miss_threshold else (1.0 - pruned_probabilities[min_idx]))
-            prob_min_ade.append(
-                min(
-                    -np.log(pruned_probabilities[min_idx]),
-                    -np.log(LOW_PROB_THRESHOLD_FOR_METRICS),
-                )
-                + curr_min_ade
-            )
-            prob_min_fde.append(
-                min(
-                    -np.log(pruned_probabilities[min_idx]),
-                    -np.log(LOW_PROB_THRESHOLD_FOR_METRICS),
-                )
-                + curr_min_fde
-            )
+            prob_min_ade.append((1 - pruned_probabilities[min_idx]) ** 2 + curr_min_ade)
```

Comment:
what was the rationale for using -log before?

maybe a reference to papers that use Brier would be useful here

Reply:
Added a reference to the original brier score in the PR description. To my knowledge, no one has used brier score with discplacent error. So there's not reference for that.
Brier score is a commonly used metric for probability calibration.